custom-words.txt

```diff
@@ -665,7 +670,6 @@ Français
 freeflow
 frontdoor
 fsaction
-Fulfilment
```

Comment:
How about this one?

Reply:
This is a typo that is suppressed on a file level in `cspell.json` and shouldn't be globally allowed